### PR TITLE
Aaronhorvath/eng 1941 export node names from skiffprosemirror

### PR DESCRIPTION
**Pull request by Aaron-EE**
Exported table node names from Prosmirror table. At the moment markdownSerializer.ts uses the DATE and LABEL node names as hard coded. After this it can import it.
Disabled trailin commas and ran prettier on the whole project to get rid of trailing commas.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -43,6 +43,15 @@ export interface CellAttributes {
   getFromDOM?: getFromDOM;
   setDOMAttr?: setDOMAttr;
 }
+export enum NodeNames {
```
Comment (amilich) [inline index.d.ts:46]: lint? 2 spaces, space after and before =, and no trailing comma on label? we could add linting to entire repo

File: src/schema/nodeNames.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export const NodeNames = {
+  TABLE: 'table',
+  TABLE_ROW: 'table_row',
+  TABLE_CELL: 'table_cell',
+  TABLE_HEADER: 'table_header',
+  CHECKBOX: 'checkbox',
+  DATE: 'date',
+  LABEL: 'label',
```
Comment (amilich) [inline src/schema/nodeNames.js:8]: nit - trailing comma?

**Review by amilich — APPROVED** [review]
LGTM! great progress

File: src/schema/nodeNames.js (hunk 2 above)
Comment (Aaron-EE) [reply src/schema/nodeNames.js]: We had trailing commas turned on in prettierrc, I have switched it to `none` and ran prettier on the repo

**Pull request merged**